packages/notification-services-controller/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/notification-services-controller",
-  "version": "0.20.1",
+  "version": "1.0.0",
```

Comment:
Can you keep this as a v0?

Reply:
There is a "breaking" peer dependency bump. But it sounds like it isn't active in the app yet? Would 0.21.0 work in that case?

Reply:
Yep perfect!

Reply:
Set it as 0.21.0